jscomp/runtime/js_typed_array.cppo.ml

```diff
@@ -180,100 +69,84 @@ end
   external byteLength : t -> int = "byteLength" [@@mel.get]\
   external byteOffset : t -> int = "byteOffset" [@@mel.get]\
   \
-  external setArray : elt array -> unit = "set" [@@mel.send.pipe: t]\
-  external setArrayOffset : elt array -> int -> unit = "set" [@@mel.send.pipe: t]\
+  external setArray : t -> elt array -> unit = "set" [@@mel.send]\
+  external setArrayOffset : t -> elt array -> int -> unit = "set" [@@mel.send]\
   (* There's also an overload for typed arrays, but don't know how to model that without subtyping *)\
   \
   (* Array interface(-ish) *)\
   external length : t -> int = "length" [@@mel.get]\
   \
   (* Mutator functions *)\
-  external copyWithin : to_:int -> t = "copyWithin" [@@mel.send.pipe: t]\
-  external copyWithinFrom : to_:int -> from:int -> t = "copyWithin" [@@mel.send.pipe: t]\
-  external copyWithinFromRange : to_:int -> start:int -> end_:int -> t = "copyWithin" [@@mel.send.pipe: t]\
+  external copyWithin : t -> to_:int -> ?start:int -> ?end_:int -> unit -> t = "copyWithin" [@@mel.send]\
   \
-  external fillInPlace : elt -> t = "fill" [@@mel.send.pipe: t]\
-  external fillFromInPlace : elt -> from:int -> t = "fill" [@@mel.send.pipe: t]\
-  external fillRangeInPlace : elt -> start:int -> end_:int -> t = "fill" [@@mel.send.pipe: t]\
+  external fill : t -> elt -> ?start:int -> ?end_:int -> unit -> t = "fill" [@@mel.send]\
   \
-  external reverseInPlace : t = "reverse" [@@mel.send.pipe: t]\
+  external reverseInPlace : t -> t = "reverse" [@@mel.send]\
   \
-  external sortInPlace : t = "sort" [@@mel.send.pipe: t]\
-  external sortInPlaceWith : (elt -> elt -> int [@u]) -> t = "sort" [@@mel.send.pipe: t]\
+  external sortInPlace : t -> t = "sort" [@@mel.send]\
+  external sortInPlaceWith : t -> f:(elt -> elt -> int [@u]) -> t = "sort" [@@mel.send]\
```

Comment:
Maybe we could consolidate these two making `f` optional?

Reply:
I didn't do this one on purpose because then you'd need the `unit` argument to `sortInPlace`. I feel like the tradeoff is justified when there are other labeled arguments but perhaps not in this case. 

If you feel strongly, we can reconsider. I think `Js.Array` and `Js.String` have similarly duplicated sorting functions.